app/models/comment.rb

```diff
@@ -1,2 +1,5 @@
 class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] as part of the requirements the Comment model should have a method that updates the comments counter for a post, kindly add that :pray: